editor/create_dialog.cpp

```diff
@@ -195,6 +195,10 @@ bool CreateDialog::_should_hide_type(const StringName &p_type) const {
 				i = script_path.find_char('/', i + 1);
 			}
 		}
+		// Abstract scripts cannot be instantiated.
+		String path = ScriptServer::get_global_class_path(p_type);
+		Ref<Script> scr = ResourceLoader::load(path, "Script");
+		return scr->is_abstract();
```

Comment:
```suggestion
		return scr.is_null() || scr->is_abstract();
```
Assuming there's no guarantee this will load to a valid result